nginx-301-https/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM nginx:alpine
+
+RUN echo 'server { \
+    listen 80; \
+    return 301 https://$host$request_uri; \
+} \
+server { \
+    listen  81; \
+    location / { \
+       return 200; \
+    } \
+}' > /etc/nginx/conf.d/default.conf
```

Comment:
I think this would be much nicer:
```
RUN echo 'server {listen 80; return 301 https://$host$request_uri;} >> /etc/nginx/conf.d/default.conf &&\
    echo 'server {listen 81; location / {return 200;}}' >> /etc/nginx/conf.d/default.conf
```

Reply:
But best would be a `COPY` instruction